pkg/assembler/graphql/examples/cve_ghsa_osv.gql

```diff
@@ -1,99 +1,92 @@
-fragment allCveTree on CVE {
+fragment allVulnerabilityTree on Vulnerability {
```

Comment:
nit: rename file to vulnerability.gql for consistency?

Reply:
done